src/views/ingresses/list/IngressesList.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { FC } from 'react';
+import { useNavigate } from 'react-router-dom-v5-compat';
+
+import {
+  IngressModel,
+  modelToGroupVersionKind,
+  modelToRef,
+} from '@kubevirt-ui/kubevirt-api/console';
+import { IoK8sApiNetworkingV1Ingress } from '@kubevirt-ui/kubevirt-api/kubernetes/models';
+import {
+  ListPageBody,
+  ListPageCreateButton,
+  ListPageFilter,
+  ListPageHeader,
+  useK8sWatchResource,
+  useListPageFilter,
+  VirtualizedTable,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { useNetworkingTranslation } from '@utils/hooks/useNetworkingTranslation';
+import useIngressColumns from '@views/ingresses/list/hooks/useIngressColumns';
+
+import IngressTableRow from './components/IngressTableRow';
+
+type IngressesListProps = {
+  kind: string;
+  namespace: string;
+};
+
+const IngressesList: FC<IngressesListProps> = ({ namespace }) => {
+  const { t } = useNetworkingTranslation();
+  const navigate = useNavigate();
+
+  const [ingress, loaded, loadError] = useK8sWatchResource<IoK8sApiNetworkingV1Ingress[]>({
+    groupVersionKind: modelToGroupVersionKind(IngressModel),
+    isList: true,
+    namespace,
+  });
+  const [data, filteredData, onFilterChange] = useListPageFilter(ingress);
+  const columns = useIngressColumns();
+
+  return (
+    <>
+      <ListPageHeader title={t('Ingresses')}>
+        <ListPageCreateButton
+          className="list-page-create-button-margin"
+          createAccessReview={{
+            groupVersionKind: modelToGroupVersionKind(IngressModel),
+            namespace,
+          }}
+          onClick={() =>
+            navigate(`/k8s/ns/${namespace || 'default'}/${modelToRef(IngressModel)}/~new`)
```

Comment:
Please check if getK8sResourcePath is suited here too, if not than please use DEFAULT_NAMESPACE

Reply:
Done!